lfucg-exactions/client/js/components/PlatZoneForm.js

```diff
@@ -153,7 +154,11 @@ function mapDispatchToProps(dispatch, props) {
                 };
                 dispatch(formUpdate(plat_zone_update));
                 if (selectedPlatZone) {
-                    const zone = activeForm.activeForm.zone;
```

Comment:
Trying to have a safety check here in case someone is pressing submit on a platzone they didn't just update. Multiple submits on a page so it could cause issues.